config/drupal-9/drupal-9.0-deprecations.php

```diff
@@ -15,14 +15,20 @@
         return new AddCommentService();
     });
 
+    if (defined('TwigSetList::TWIG_24')) {
+        $twigSet = TwigSetList::TWIG_24;
+    } else {
+        $twigSet = TwigSetList::TWIG_240;
+    }
```

Comment:
@bbrala 
This does not work for me, I needed whole namespace, there is same issue in `config/drupal-10/drupal-10.0-deprecations.php`
```suggestion
    if (defined('\Rector\Symfony\Set\TwigSetList::TWIG_24')) {
        $twigSet = TwigSetList::TWIG_24;
    } else {
        $twigSet = TwigSetList::TWIG_240;
    }
```